Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:16-alpine
+
+WORKDIR /app
+
+COPY . /app
```

Comment:
It's usually a good pattern to copy only the package json files first, install dependencies and then copy the rest of the app.
```
COPY package*.json /app
RUN npm ci
COPY . /app
```
This speeds up builds because package json is less likely to change then the rest of the app so docker can use cached image layers.

Reply:
Fixed.